assets/global.js

```diff
@@ -1110,6 +1110,12 @@ class VariantSelects extends HTMLElement {
         const pricePerItemSource = html.getElementById(`Price-Per-Item-${this.dataset.originalSection ? this.dataset.originalSection : this.dataset.section}`);
 
         const volumePricingDestination = document.getElementById(`Volume-${this.dataset.section}`);
+        const qtyRules = document.getElementById(`Quantity-Rules-${this.dataset.section}`);
+        const volumeNote = document.getElementById(`Volume-Note-${this.dataset.section}`);
+
+        if (volumeNote) volumeNote.classList.remove('visibility-hidden');
+        if (volumePricingDestination) volumePricingDestination.classList.remove('visibility-hidden');
```

Comment:
I think it's better to replace `visibility-hidden` with `hidden`. As far as I understand we want to keep layout only above input picker, so I would prefer to hide volume pricing table and not to keep the space. 
<img width="641" alt="Screenshot 2023-07-21 at 9 27 44 AM" src="https://github.com/Shopify/dawn/assets/105315663/fce8972f-563c-4e9d-acde-3d36e67538b9">

cc: @edmund-teh

Reply:
> I think it's better to replace `visibility-hidden` with `hidden`. As far as I understand we want to keep layout only above input picker, so I would prefer to hide volume pricing table and not to keep the space. <img alt="Screenshot 2023-07-21 at 9 27 44 AM" width="641" src="https://user-images.githubusercontent.com/105315663/255230426-fce8972f-563c-4e9d-acde-3d36e67538b9.png">
> 
> cc: @edmund-teh

You make a good point @eugenekasimov! Plus the `Unavailable` button is disabled so a buyer won't be able to take any action here. We also wanted to avoid the jumping with the different variant options, which we still would be doing.

Reply:
Since `sku` and `price per item` are using `visibility hidden` I think we should follow the same for now for consistency.

We can address this issue when we address the existing spacing issue cc @edmund-teh - i believe we have an issue for this? cant seem to find it

Reply:
This issue? @sofiamatulis #2870?

Reply:
@edmund-teh I thought we had an issue for the extra spacing? 🤔

Reply:
Let's track it separately 👍